scripts/store/configureStore.js

```diff
@@ -14,6 +14,7 @@ const finalCreateStore = compose(
 
 export default function configureStore(initialState) {
   const store = finalCreateStore(rootReducer, initialState);
-  sagaMiddleware.run(rootSaga);
+  const getState = store.getState.bind(store);
```

Comment:
Add a comment explaining that we want all saga to have access to the store state so we bind it to the rootSaga.